### PR TITLE
config.py-removed mysql param

### LINKED ISSUE #4: [rabbit-to-db] Create docker image
Please add it to public.ecr.aws/b4p0m4d6/kv126 (see https://eu-central-1.console.aws.amazon.com/ecr/repositories/public/779414916509/kv126?region=eu-central-1)

There should be 2 versions `latest` and `1.0.0`

### PR DESCRIPTION
**Pull request by Olyakaya**
Added Dockerfile

### CODE REVIEW

**Comment by Olyakaya** [comment]
incorrect

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+FROM python:3.9-slim
+
+RUN useradd -ms /bin/bash kvuser
+USER kvuser
+WORKDIR /home/kvuser
+
+ENV PATH="/home/kvuser/.local/bin:${PATH}"
+
+COPY --chown=kvuser:kvuser requirements.txt requirements.txt
+RUN python3 -m pip install --upgrade pip
```
Comment (anatolek) [inline Dockerfile:10]: This uses only extra space in the image and in the future may lead to breakage
```suggestion
```

**Review by anatolek — CHANGES_REQUESTED** [review]

**Comment by Olyakaya** [comment]
removed string RUN python3 -m pip install --upgrade pip
recreated and uploaded to the ECR new image

File: Dockerfile (hunk 1 above)
Comment (Olyakaya) [reply Dockerfile:10]: There was removed string RUN python3 -m pip install --upgrade pip
recreated and uploaded to the ECR new image

**Review by anatolek — APPROVED** [review]

**Pull request merged**